File: Python/Arithmetic.py
import random;


print("solve this subtraction equation")

count = 0

for counter in range(1,11):
    first_digit = random.randint(100,200)
    second_digit = random.randint(1,99)

#    if second_digit > first_digit:
#        subtraction = second_digit - first_digit 
#        print(second_digit, "-", first_digit)
#    elif:
    subtraction = first_digit - second_digit
    
    print(first_digit, "-", second_digit)

        answer = input("Enter an answer: ")
    
    if (answer == subtraction):
         print("You are correct")
         count += 1     
    elif(answer != subtraction):       
        print("You are incorrect")
        answer = input("Enter an answer: ")

if count > 5:
    print("your final score is: ", count , "impressive!!!") 
else:
    print("your final score is: ", count, "poor!!!")   
    



    
   

File: Java/Arithmetic.java
import java.util.Random;
import java.util.Scanner;


public class Arithmetic{
public static void main(String[] args){
Scanner input = new Scanner(System.in);
Random random = new Random();

System.out.println("Solve this subtraction equation");

int count = 0;

for(int counter = 1; counter <= 10; counter++){
int subtraction;
int firstDigit = random.nextInt(1,100);
int secondDigit = random.nextInt(1,100);

if (secondDigit > firstDigit){
subtraction = secondDigit - firstDigit;
 System.out.println(secondDigit + "-" + firstDigit);
}else{
subtraction = firstDigit - secondDigit;
 System.out.println(firstDigit + "-" + secondDigit);
}


System.out.println("Enter answer: ");
int answer = input.nextInt();

if(answer == subtraction){
System.out.println("you are correct");
count += 1;
}else{
System.out.println("incorrect");
System.out.println("Enter answer: ");
 answer = input.nextInt();
}

}


if(count > 5){
System.out.println("Your final score is: " + count + " Superb!!!");
}else{
System.out.println("Your final score is: " + count + " poor!!!,you can do better");
}





}
}

